cxmon/src/mon_cmd.cpp

```diff
@@ -32,6 +33,9 @@
 #endif
 
 
+static const char* STR_ACTIVE_BREAK_POINTS 	= "Active Break Points:\n";
```

Comment:
Nit: const char STR_ACTIVE_BREAK_POINTS[] =

I believe this results in more efficient code with some compilers.

Reply:
Done!